### PR TITLE
Added Travis Settings

### LINKED ISSUE #71: No Travis Settings
Travis Settings are missing

### PR DESCRIPTION
**Pull request by NaviRocker**
Fixes #71 
Added Travis Settings

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+before_install:
+  - gem install github-pages --no-rdoc --no-ri
+
+rvm:
+- 2.1
+script: jekyll build
+
+branches:
+  only:
+  - gh-pages
```
Comment (abishekvashok) [inline .travis.yml:12]: Add other branches also

**Review by abishekvashok — CHANGES_REQUESTED** [review]

File: .travis.yml (hunk 1 above)
Comment (NaviRocker) [reply .travis.yml:12]: Done

**Comment by abishekvashok** [comment]
Can you squash as well?

**Comment by NaviRocker** [comment]
Squashed the Commits

**Review by isuruAb — APPROVED** [review]
👍

**Pull request merged**